styles.css

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0.05em;
+  padding: 0.05em;
+  box-sizing: border-box;
+  font-size: 20px;
+}
+
+body {
+  /* background: #ffff; */
+  background: #e5e5e5;
+  min-height: 100vh;
+}
+
+.container {
+  display: flex;
+  background: #ffff;
+  flex: 1;
+  border-radius: 0.05vw 0.05vw 0.05vw 30vw;
+  flex-direction: column;
+  background-image: url("images/header-shapes-mobile.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: absolute;
+  width: 100%;
+  min-height: 100vh;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+.topnav {
+  align-items: center;
+  padding: 0.2em;
+}
+
+.navi ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-size: 18px;
+  padding: 0.5rem;
+}
+
+.navi a {
+  text-decoration: none;
+  color: #6070ff;
+  font-weight: 700;
+}
+
+.logo img {
+  border: 2px solid;
+  height: 60px;
+  border-radius: 50%;
+}
+
+.intro {
+  margin-top: 206px;
+  padding: 20px;
+}
+
```

Comment:
- According to the project requirements, you are required to use the `flex` display property to align this section. This would help improve the responsiveness of your site. Please fix this 🙏